core/templates/dev/head/pages/forum/Forum.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Data and controllers for the Oppia 'forum' page.
+ */
+
+oppia.controller('Forum', [
+  '$scope',
+  function(
+      $scope) {
```

Comment:
No need to move these in newlines, as there are very few arguments, can keep it as 
`'$scope', function($scope) {`.

Reply:
Done !!